R/use_ensemble_criteria.R

```diff
@@ -72,7 +72,7 @@ use_ensemble_criteria <- function(forecasts,
 
   # 5. Drop "other" designated models
   if (exclude_designated_other) {
-    not_other <- get_model_designations(here()) %>%
```

Comment:
Do you run into any problems with this? Prefixing it shouldn't be necessary within the package.

Reply:
Yep, probably my mistake, ignore that diff.